### PR TITLE
Copy edits and MS comments: Part 1

### PR DESCRIPTION
**Pull request by sjspielman**
This PR does some copy editing and adds in a lot of comments marked as **TODO** that can be addressed during review for this PR. 

Specifically, this PR considers the front matter, introduction, and about the first half of the results section; I stopped for now at the beginning of `Prediction of _TP53_ oncogenicity and telomerase activity` while we have further discussion at Issue #261.

(Note I have a commit in there that says "summary is not 150 words" but is should say "summary is noW 150 words"!! big difference!).

Edit: Forgot to add these additional comments:

+ Numbers are a little all over the place and hard to follow re-how many samples there are. **Edit:** This is now tracked [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1392)
  + Highlights says: 1253 tumors and 2840 biospecimens
  + Introduction says:  2,000 tumor specimens and 38 cell lines, comprised of 59 distinct brain tumor histologies from 943 patients. 
  + Introduction says: 1,281/_2,007_ were subtyped
  + Results says: "We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across **1,969 tumors** (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.

+ Some text I tried to clean up but more eyes would be good:
   + Introduction opens with declaring these tumors are the 2nd most common cause of death, but then continues to state they are rare. This is confusing
  + The Introduction part about the RACE Act is a not clearly organized. For example, if we are contending genomics is moving fast and the problem is bureaucracy, then how is more genomics going to support RACE?	
+  Figure 1B is not mentioned in the text so I've tried to add it in

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
-There has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, however, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approval has been slow. 
+Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace. 
 Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas.
```
Comment (sjspielman) [inline content/02.introduction.md:14]: TODO: This needs a reference of some kind.

File: content/00.front-matter.md [hunk 2]
```diff
@@ -61,16 +61,18 @@ on {{manubot.date}}.
...
+The OpenPBTA framework can be used as a model for large-scale data integration to inform basic research, therapeutic target identification, and clinical translation. 
 
 ## Highlights
 <!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
 OpenPBTA collaborative analyses establish resource for 1,253 pediatric brain tumors
 
 NGS-based WHO-aligned integrated diagnoses generated for 1,260 of 2,840 biospecimens
 
-Classifiers identify medulloblastoma subgroups, TP53 status, and telomerase activity
+Classifiers identify medulloblastoma subgroups and TP53 status, and telomerase activity [TODO: NOT A CLASSIFIER!!!!]
```
Comment (sjspielman) [inline content/00.front-matter.md:74]: I thought this was previously fixed in a different PR but I guess not...

File: content/02.introduction.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-For example, most high-grade and embryonal tumors carry a universally fatal prognosis while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
-Despite their relative rarity, the years of potential life lost due to brain tumors in 2009 was estimated at 47,631 years for children and adolescents aged 0-19 in the United States [@doi:10.1159/000093896]. 
-The low survival rates for some tumors are clearly multifactorial but can be explained in part by our lack of understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these entities, and the shortage of drugs specifically labeled for pediatric malignancies. 
+For example, most high-grade and embryonal tumors carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
+Moreover, estimates from 2006 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1159/000093896].
```
Comment (sjspielman) [inline content/02.introduction.md:6]: I checked this citation, it's from 2006

File: content/03.results.md [hunk 4]
```diff
@@ -2,41 +2,48 @@
...
+We previously (**TODO: looks like this "previously" was only pHGG?**) performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**). 
+We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses including variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
+A summarized view of the number of biospecimens per phase of therapy across different histology groupings is shown in (**Figure {@fig:Fig1}B**).
```
Comment (sjspielman) [inline content/03.results.md:7]: Added this sentence in so Fig1B is referenced in the text.

File: content/03.results.md [hunk 5]
```diff
@@ -2,41 +2,48 @@
...
+We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses including variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
+A summarized view of the number of biospecimens per phase of therapy across different histology groupings is shown in (**Figure {@fig:Fig1}B**). 
+We maintained a data release folder on Amazon S3, downloadable using the GitHub repository or the open-access [CAVATICA project](https://www.cavatica.org/), containing merged files for each analysis (See **Data and code availability** section). **TODO: this is mentioned before mention the repo itself!**
```
Comment (sjspielman) [inline content/03.results.md:8]: The TODO here is organizational. We can either talk about the repo earlier, or not talk about the repo in this sentence at all.

File: content/03.results.md [hunk 6]
```diff
@@ -2,41 +2,48 @@
...
-**Table 1** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). 
+Over the past two decades, experts in neurooncology have worked with the World Health Organization (WHO) to iteratively redefine the classifications of central nervous system (CNS) tumors [@doi:10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
+More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these classifications. 
+In 2011, the Children’s Brain Tumor Tissue Consortium (CBTTC), now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
```
Comment (sjspielman) [inline content/03.results.md:41]: "opened its protocol" is not a phrase I am familiar with, so I've assumed it's a clinical phrase I don't know and did not modify it. Thoughts?

File: content/03.results.md [hunk 7]
```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
 We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### Diffuse astrocytic and oligodendroglial tumors
-In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
-The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
-Across HGGs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
+In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26) biopsied at diagnosis.
```
Comment (sjspielman) [inline content/03.results.md:138]: I think we might want to add a sentence plainly stating there are N=61 total, so the denominators in the rest of the paragraph have a clear meaning.

File: content/03.results.md [hunk 8]
```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
 We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
-Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
-The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
+Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**). **TODO: if we want to reference a figure 5 panel _here_, we need to moe the panel. Otherwise figures are out of order.**
```
Comment (sjspielman) [inline content/03.results.md:142]: (move not moe)

File: content/03.results.md [hunk 9]
```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
+**TODO: A lot of different denominators here with no clear meaning. What is 666 vs 872 vs 54?**
+
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). **TODO: Clarify more what "top 50" means. Most mutations? Most commonly mutated across samples? etc.**
+_TP53_ was the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
+Other canonical cancer driver genes that were frequently mutated included _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
+Although LGG and embryonal tumors made up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stemmed from HGGs (N = 847/872). **TODO: Is this denominator the total number of significant? Unclear what these numbers are**
```
Comment (sjspielman) [inline content/03.results.md:163]: woops ,put this TODO in twice! focus on the one at the top of this section, line 158.

File: content/03.results.md [hunk 10]
```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
 
 ### Transcriptomic Landscape of Pediatric Brain Tumors
 
 #### Prediction of _TP53_ oncogenicity and telomerase activity
 
-To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
+To understand the _TP53_ phenotype in each tumor, we applied a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
```
Comment (sjspielman) [inline content/03.results.md:190]: note this was my only edit towards this section of the manuscript. This will be in my next round.

File: content/03.results.md (hunk 8 above)
Comment (jharenza) [reply content/03.results.md:142]: I don't necessarily think we need to move them per se - this is a challenge because GSVA is really a functional result of the somatic alterations, so it makes sense to back up the oncoprints with this data (I was asked about this for the PPTC paper before submitting), but GSVA can also stand alone with additional findings. So, it is tough - some people make some pie or bar plots with pathways or genes upregulated to go along with oncoprints, but it is hard to squeeze everything in, so this is what I had defaulted to.

File: content/03.results.md [hunk 11]
```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
+Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**). **TODO: if we want to reference a figure 5 panel _here_, we need to moe the panel. Otherwise figures are out of order.**
+The two tumors with ultra-high TMB (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
 
 #### Other CNS tumors
-**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
-We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
+**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors in the OpenPBTA (N = 195). 
+We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas and that 70% (21/30) of craniopharyngiomas were driven by mutations in _CTNNB1_.
```
Comment (jharenza) [inline content/03.results.md:147]: grant writing tricks - removing extra words ("that") 😄  - we also will need to fall within a certain word count, so may need to remove some of this later, but I haven't even checked this yet!

File: content/03.results.md (hunk 9 above)
Comment (jharenza) [reply content/03.results.md:163]: Now I am second guessing myself:
- 666 is the total N of tumors in the co-occurrence module (across all PBTA) from sum of Ns in rows `cooccur_top50.ALL.tsv`
- Seems like `847/872` comes from:
```r
co <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/interaction-plots/results/cooccur_top50.ALL.tsv") %>%
  filter(q < 0.05)
```
This has nrow = 872 significant events
```r
hgg <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/interaction-plots/results/cooccur_top50.Diffuse-astrocytic-and-oligodendroglial-tumor.tsv") %>%
  filter(q < 0.05)
```
This has nrow 847

File: content/03.results.md (hunk 10 above)
Comment (jharenza) [reply content/03.results.md:190]: I don't see anything from line 164-190, aside from this, is that correct?

**Review by jharenza — CHANGES_REQUESTED** [review]
thanks @sjspielman ! looking good so far - just a few comments

File: content/01.summary.md [hunk 12]
```diff
@@ -1,13 +1,14 @@
...
-We establish over 40 scalable analysis modules, recapitulate key oncogenic drivers across histologies, and make the code and data openly available.  
-The OpenPBTA has become a foundational analysis platform that is actively being applied to study other pediatric cancers and inform molecular tumor board decision-making, making it an invaluable resource to the pediatric oncology community. 
-s
+<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 150 -->
+Pediatric brain and spinal cord tumors are the leading disease-related cause of death in children, and novel, curative therapeutic strategies for these tumors are urgently needed.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/01.summary.md:3]: grammatically this would need a semicolon since "thus" can't be used to form a compound sentence. - "Pediatric brain and spinal cord tumors are the leading disease-related cause of death in children **;** thus, novel, curative therapeutic strategies for these tumors are urgently needed."

File: content/01.summary.md [hunk 13]
```diff
@@ -1,13 +1,14 @@
...
+To accelerate such discoveries, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium created a systematic process for tumor biobanking, model generation, and sequencing and provided immediate access to harmonized data.
+Here, we leverage these data to create the OpenPBTA project to openly and collaboratively genomically characterize and molecularly subtype, to our knowledge, the largest cohort of pediatric brain tumor specimens and histologies to date.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/01.summary.md:5]: In the interest of <150 word count, how about this - 
"Here, we leverage these data to create the OpenPBTA project which openly and collaboratively genomically characterizes and molecularly subtypes, to our knowledge, the largest cohort of pediatric brain tumor specimens and histologies to date"

[the remark this answers is not in the document]
Comment (sjspielman) [reply content/01.summary.md:5]: Actually, since the next sentence _also_ uses openly, we can cut words like this:

Here, we leverage these data to create the OpenPBTA project in which we openly and collaboratively genomically characterize and molecularly subtype, to our knowledge, the largest cohort of pediatric brain tumor specimens and histologies to date.

The OpenPBTA establishes over 40 scalable analysis modules to analyze these data and recapitulate key oncogenic drivers across histologies.

File: content/02.introduction.md (hunk 3 above)
Comment (sjspielman) [reply content/02.introduction.md:6]: I actually wonder if this is the citation? https://onlinelibrary.wiley.com/doi/full/10.1002/cam4.410

I don't have full access text to thoroughly check though.

File: content/02.introduction.md [hunk 14]
```diff
@@ -1,18 +1,19 @@
...
-Despite their relative rarity, the years of potential life lost due to brain tumors in 2009 was estimated at 47,631 years for children and adolescents aged 0-19 in the United States [@doi:10.1159/000093896]. 
-The low survival rates for some tumors are clearly multifactorial but can be explained in part by our lack of understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these entities, and the shortage of drugs specifically labeled for pediatric malignancies. 
+For example, most high-grade and embryonal tumors carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
+Moreover, estimates from 2006 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1159/000093896]. 
+Among the key drivers of low survival rates for some pediatric tumors are a lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs indicated for pediatric malignancies.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/02.introduction.md:7]: Totally fair. But, I was very thrown off by the phrase "clearly multifactorial" though which seems overly vague and making lots of assumptions that are never spelled out. Directly listing out some reasons for low survival rates, with maybe a review paper citation, would help.

File: content/03.results.md [hunk 15]
```diff
@@ -2,41 +2,48 @@
...
+A summarized view of the number of biospecimens per phase of therapy across different histology groupings is shown in (**Figure {@fig:Fig1}B**). 
+We maintained a data release folder on Amazon S3, downloadable using the GitHub repository or the open-access [CAVATICA project](https://www.cavatica.org/), containing merged files for each analysis (See **Data and code availability** section). **TODO: this is mentioned before mention the repo itself!**
+As new analytical results (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses were produced, or issues with the data were identified, new data releases were made available in a versioned manner.
 
-A key innovative feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
+A key innovative feature of this project was its open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/03.results.md:11]: Maybe we use "has been" here, since it's past but ongoing to some degree

File: content/01.summary.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply content/01.summary.md:3]: That sounds better!

File: content/01.summary.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply content/01.summary.md:5]: >Here, we leverage these data to create the OpenPBTA project in which we openly and collaboratively genomically characterize and molecularly subtype, to our knowledge, the largest cohort of pediatric brain tumor specimens and histologies to date.

>The OpenPBTA establishes over 40 scalable analysis modules to analyze these data and recapitulate key oncogenic drivers across histologies.

👍

File: content/02.introduction.md (hunk 3 above)
Comment (jharenza) [reply content/02.introduction.md:6]: YES, you got it! From abstract: 0.31*153,390.4 = 47,551, so we should update.

Comment (jharenza) [reply content/02.introduction.md:6]: Didn't seem to have a paywall for me. Actually, Table 2 is where the `47,631.5` comes from:
Screen Shot 2022-05-09 at 2 15 05 PM

File: content/02.introduction.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply content/02.introduction.md:7]: The reasons are spelled out: part by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies.

Am I missing something?

File: content/02.introduction.md (hunk 3 above)
Comment (sjspielman) [reply content/02.introduction.md:6]: Great! Will update accordingly!!

File: content/02.introduction.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/02.introduction.md:7]: Aha, I see it now. The "but" made me think this phrase was aside from additional reasons. If I change the "but can be explained" to "_and_ can be explained," it flows better as the explanation - 
The low survival rates for some pediatric tumors are clearly multifactorial **and** can partly be explained by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies.

File: content/03.results.md [hunk 16]
```diff
@@ -2,41 +2,48 @@
...
-We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or the open-access CAVATICA project (See **Data and code availability** section).
-As new analytical products (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses were produced, or issues with the data were identified, new releases were made available in a versioned manner.
+We previously (**TODO: looks like this "previously" was only pHGG?**) performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**).
```
Comment (sjspielman) [inline content/03.results.md:5]: @jharenza Can you comment on this TODO? I was confused because the PBTA citation looks like it was only 37 samples, not 943.

Comment (jharenza) [reply content/03.results.md:5]: The citation is for the cell lines, specifically

File: content/03.results.md (hunk 5 above)
Comment (sjspielman) [reply content/03.results.md:8]: I'm rephrasing this to remove a reference to the repo itself, and say you can download it directly from S3 itself. The repo contains a script that does just that, so the overall meaning should be the same.

File: content/03.results.md (hunk 16 above)
Comment (sjspielman) [reply content/03.results.md:5]: Ahhhh!!! 👍

File: content/03.results.md [hunk 17]
```diff
@@ -2,41 +2,56 @@
...
+We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses including variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
+To facilitate analysis and visualization of this large, diverse cohort, we further categorized tumor broad histologies (i.e., broad 2016 WHO classifications) into smaller groupings we denote "cancer groups."
+A summarized view of the number of biospecimens per phase of therapy across different broad histologies and cancer groups is shown in (**Figure {@fig:Fig1}B**).
```
Comment (sjspielman) [inline content/03.results.md:8]: @jharenza I updated this part and also modified the text in the 1B legend as a way to communicate cancer group/broad histology terminology early in the paper.

File: content/03.results.md (hunk 11 above)
Comment (sjspielman) [reply content/03.results.md:155]: Counterpoint: "that...were" twice is the same number of words as "to be" twice ;)

File: content/03.results.md (hunk 9 above)
Comment (sjspielman) [reply content/03.results.md:171]: Ok, we should probably add a (likely extremely small!) notebook to the module then to calculate these numbers. They need to have a direct source in the repo.

File: content/03.results.md [hunk 18]
```diff
@@ -114,69 +129,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
+Signature N6 is a CNS-specific signature which we observed nearly universally across samples.
+Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observed Signature 18 as the most common signature in medulloblastoma tumors.
+Signatures 3, 8, 18, and MMR2 were prevalent in HGGs, including DMGs.
+Finally, we found that the exposure to Signature 1 was higher at diagnosis (pre-treatment) and was almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy; **Figure {@fig:S4}B**).
+This trend may have resulted from therapy-induced mutations that produced additional signatures (e.g., temozolomide treatment drives Signature 11 **TODO: this is not a strong claim; the refsig website says this finding is not reproducible.**), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.
```
Comment (sjspielman) [inline content/03.results.md:188]: Note I'm deleting the temozolomide parenthetical. This was the reference which explains there is no strong evidence for this association - https://signal.mutationalsignatures.com/explore/referenceCancerSignature/40?studyId=1

> This signature was previously associated with temozolomide exposure, having been first identified in patients that had temozolomide during treatment of their intracranial tumours. However, the signature has not been recapitulated in experimental model systems. In spite of multiple independent experiments with temozolomide exposure, the signature was consistently different (see Temozolomide 200 µM (A) Substitution Signature). Intriguingly, the signature of treatment with 1,2-DMH was found to be very similar to this signature (see 1,2-Dimethylhydrazine 11.6 mM Rat S9 Rat S9 Substitution Signature).

File: content/03.results.md (hunk 10 above)
Comment (sjspielman) [reply content/03.results.md:198]: Yes that's it for this PR!

File: content/03.results.md (hunk 18 above)
Comment (jharenza) [reply content/03.results.md]: I suggest we instead write, "suggested to drive". I do understand that _one_ paper shows that this signature isn't TMZ, but model systems are not perfect. The paper says they used `human-induced pluripotent stem cells`, but perhaps they are not recapitulating brain cell diversity and the mutational signatures are not recapitulated perfectly. I looked at [this paper](https://pubmed.ncbi.nlm.nih.gov/25088490/v) to learn a bit more about DMH (I can send you if you would like). It states that 
>A methylating agent that preferentially targets the colon is 1,2-dimethylhydrazine (DMH). 

So if the COSMIC Sig 11 is due to DMH, which targets the colon, it wouldn't make sense for that signature to be in brain tumor samples at all. I am just not wholly convinced that Sig 11, which is observed in the samples we know were treated with TMZ, is not associated with TMZ exposure. 

I guess I am a bit weary of one pub challenging _years_ of "previously hypothesized truth" (not saying it isn't possible or isn't real), but it might be contextual (in this model, it produces a signature like this) or more complex if perhaps signature 11 is due to TMZ+radiation for example, but not TMZ or radiation alone, and they didn't test combinations. So I would feel more comfortable acknowledging this paper exists and says one thing contrary to all other known literature, but still acknowledging that this is suggested as a mechanism driving Sig 11.

File: content/03.results.md (hunk 11 above)
Comment (jharenza) [reply content/03.results.md:155]: ha!!

File: content/03.results.md (hunk 17 above)
Comment (jharenza) [reply content/03.results.md:8]: great! thank you

File: content/03.results.md (hunk 8 above)
Comment (sjspielman) [reply content/03.results.md]: For now I'm going to remove this comment but open an issue so we can separately discuss this figure organization aspect.

File: content/03.results.md [hunk 19]
```diff
@@ -114,69 +129,74 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
-Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
-Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure {@fig:S4}B**.
-This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.
+**TODO: A lot of different denominators here with no clear meaning. What is 666 vs 872 vs 54?**
+
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). **TODO: Clarify more what "top 50" means. Most mutations? Most commonly mutated across samples? etc.**
```
Comment (sjspielman) [inline content/03.results.md:168]: @jashapiro can you speak to this TODO item regarding the specific meaning of "top 50"? Thanks!

File: content/03.results.md [hunk 20]
```diff
@@ -114,69 +129,74 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
...
+
+We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medulloblastomas, had the most unstable genomes (**Figure {@fig:S3}A**).
+By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
+Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
+The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
+We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas and in 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
+We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors. **TODO: 15% of EACH of these groups, or TOTAL 15%?**
```
Comment (sjspielman) [inline content/03.results.md:181]: I see these numbers are also from 3D. I'm going to rearrange this text so that it matches the order and terminology used in 3D.

File: content/03.results.md (hunk 19 above)
Comment (jashapiro) [reply content/03.results.md:167]: This is the set of genes that are mutated in the greatest number of samples. If a sample has more than one mutation in a gene, it is still only counted once.

Some more detail about what defines a countable mutation is at https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#recurrently-mutated-genes-and-co-occurrence-of-gene-mutations, though that should be updated to add that we exclude frequently mutated genes from non-tumor genomes, specifically the top 50 FLAGS genes (from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706417/bin/12920_2017_309_MOESM3_ESM.txt, which is the corrected data for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4267152/)

Comment (sjspielman) [reply content/03.results.md:167]: Ok, I'll just refer to the Methods here for an explanation of what top 50 refers to.

File: content/03.results.md (hunk 9 above)
Comment (jharenza) [reply content/03.results.md]: PR: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1398

**Comment by jharenza** [comment]
@sjspielman do you want to merge with the TODOs or get those settled before merge? I think you can probably put HTML tags in instead if we merge before they are settled and that way, they don't render on the web

**Comment by sjspielman** [comment]
@jharenza all the TODOs are gone at this point, except for one which is in an HTML comment. At this point it seems everything is settled to me, and we can catch any small foibles in future rounds with more eyes on the MS.

**Review by jharenza — APPROVED** [review]
